dotcom-rendering/src/web/components/NewslettersBadge.tsx

```diff
@@ -0,0 +1,65 @@
+import { css } from '@emotion/react';
+import { brandAlt, headline, neutral } from '@guardian/source-foundations';
+
+const textStyle = (textColour: string) => css`
+	${headline.small({ fontWeight: 'bold' })};
+	font-size: 19.5px;
```

Comment:
```suggestion
```

As above, we can use the `headline` api to set this. Note. I think decimals in px are valid but for consistency I guess best avoided? Consistent font sizing when reading text is relaxing... 🧘

Reply:
I know this is pretty horrible, but because the style is being applied to a`<text>` within an `<svg>`, the size is relative to SVG's coordinate system, and 19.5px is the size I need to make it match the design - it equates to about 35px text in  the document.
I'll put some comments in about that.

Reply:
I went away and checked this with a few people (because I really wasn't sure myself). The advice I've got is to never use sub pixel values. The reasons for this are:

- Browsers will round up and down inconsistently, so things will look different in different contexts
- You can end up with antialiasing or blurriness on different screen types (non retina)

Reply:
It's probably best not to use pixels (let alone sub pixels) for font-size within an SVG as its misleading to have css where "px"≠1px  - I've changed this to a % value for the style applied to the svg `text`